src/shared/hooks/DateFunctions.ts

```diff
@@ -16,4 +16,19 @@ export function getTime(dateTime: string): string {
 export function getEndTime(dateTime: string): string {
     const fullTime = getTime(dateTime)
     return (parseInt(fullTime) === 0 ? "24" : fullTime)
+}
+
+export function getDateNumber(dateTime: string): number {
```

Comment:
Counting everything together won't always give the correct result when you are going to sort with it.